package.json

```diff
@@ -1,5 +1,10 @@
 {
+  "scripts": {
+    "upload": "bash upload.bash"
+  },
   "devDependencies": {
+    "jq": "^1.7.2",
```

Comment:
This is the wrong dependency right? Should be node-jq ?

Reply:
I thought so. Weird things are happening today.